proto/trainer_client.proto

```
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package trainer;
```

Buf (1.54.0)

Files with package "trainer" must be within a directory "trainer" relative to root but were in directory "proto".

(PACKAGE_DIRECTORY_MATCH)